CHANGELOG.md

```diff
@@ -73,6 +73,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Crash when trying to start on X11 with a Wayland compositor running
 - Crash with a virtual display connected on X11
 - Use `\` instead of `\\` as path separators on Windows for logging config file location
+- Underline/strikeout drawn above visual bell
+- Terminal going transparent during visual bell
+- Visual bell being above message bar background
```

Comment:
This last one (visual bell above message bar) is not actually an issue on the latest master. I think it was just temporarily introduced while working on it.